tests/runner/kube_testplatform.go

```diff
@@ -43,11 +45,33 @@ func (c *KubeTestPlatform) tearDown() error {
 		log.Errorf("Failed to tearDown AppResources. got: %q", err)
 	}
 
+	if err := c.ComponentResources.tearDown(); err != nil {
+		log.Errorf("Failed to tearDown ComponentResources. got: %q", err)
+	}
+
 	// TODO: clean up kube cluster
 
 	return nil
 }
 
+// addComponents adds component to disposable Resource queues
+func (c *KubeTestPlatform) addComponents(comps []kube.ComponentDescription) error {
+	if c.kubeClient == nil {
+		return fmt.Errorf("kubernetes cluster needs to be setup")
+	}
+
+	for _, comp := range comps {
+		c.ComponentResources.Add(kube.NewDaprComponent(c.kubeClient, kube.DaprTestKubeNameSpace, comp))
```

Comment:
Will kube.NewDaprComponent(c.kubeClient, kube.DaprTestKubeNameSpace, comp) result in error ?

Reply:
No it will never return error